internal/server/db/backups.go

```diff
@@ -38,6 +38,16 @@ type StoragePoolVolumeBackup struct {
 	CompressionAlgorithm string
 }
 
+// StoragePoolBucketBackup is a value object holding all db-related details about a storage bucket backup.
```

Comment:
I'd put the `internal/server/db` changes into a new `incusd/db: Add storage bucket backup functions`